src/l500/l500-options.cpp

```diff
@@ -164,6 +165,12 @@ namespace librealsense
         case RS2_L500_VISUAL_PRESET_CUSTOM:
             move_to_custom();
             break;
+        case RS2_L500_VISUAL_PRESET_DEFAULT:
+            LOG_ERROR("changing back to default is disabled by design");
```

Comment:
Need to give context: in the log, someone will not understand what you're talking about